dicoogle/src/main/java/pt/ua/dicoogle/server/TransfersStorage.java

```diff
@@ -47,6 +47,7 @@
      */
     
     public static final Map<Integer, String> globalTransferMap;
+    public static final Map<String, String> globalTransferUIDsMap;
```

Comment:
Odd, I would expect this dictionary to already exist in dcm4che... Would [`UIDDictionary`](https://github.com/dicomgrid/dcm4che2/blob/master/dcm4che-core/src/main/java/org/dcm4che2/data/UIDDictionary.java) work here?

Reply:
No. This dictionary does not work in the way that I need. It returns a different UID description.